instructeurs/models.py

```diff
@@ -33,6 +33,7 @@ class Administration(IngestableModel):
     prefix_convention = models.CharField(
         max_length=255, default="{département}/{zone}/{mois}/{année}/80.416/", null=True
     )
+    signature_label_extra = models.TextField(null=True)
```

Comment:
To be able to save it without value in admin

```suggestion
    signature_label_extra = models.TextField(null=True, blank=True)
```